cms/envs/production.py

```diff
@@ -242,6 +243,52 @@ def get_env_setting(setting):
 # by end users.
 CSRF_COOKIE_SECURE = ENV_TOKENS.get('CSRF_COOKIE_SECURE', False)
 
+# Determines which origins are trusted for unsafe requests eg. POST requests.
+CSRF_TRUSTED_ORIGINS = ENV_TOKENS.get('CSRF_TRUSTED_ORIGINS', [])
+
+############# CORS headers for cross-domain requests #################
+
+if FEATURES.get('ENABLE_CORS_HEADERS') or FEATURES.get('ENABLE_CROSS_DOMAIN_CSRF_COOKIE'):
```

Comment:
So, looking our internal configuration, this will evaluate to `True` and enable the usage of a bunch of Django settings that, as far as I can tell, have never been used in Studio. There's nothing wrong with that *per se* but I'd like to do some more internal investigation before giving the CORS/CSRF settings my OK.

Reply:
This is now moot, as CORS support was merged via https://github.com/edx/edx-platform/pull/24571.